src/scene/graphics/scene-grab.js

```diff
@@ -54,6 +54,18 @@ class SceneGrab {
         // create depth layer
         this.layer = null;
 
+        // null device does not support scene grab
+        if (this.device.isNull) {
```

Comment:
Instead of needing a property for `isNull`, could you just do:

```javascript
if (this.device instanceof NullGraphicsDevice) {
```

Reply:
but that requires an import, adding a dependency - and we would not be able to tree-shake the null device.

Reply:
OK, fair enough.